sdk/storage/Microsoft.Azure.WebJobs.Host.EndToEndTests/tests/BlobTriggerEndToEndTests.cs

```diff
@@ -19,7 +20,7 @@
 
 namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
 {
-    public class BlobTriggerEndToEndTests : IDisposable
+    public class BlobTriggerEndToEndTests : WebJobsTestBase, IDisposable
```

Comment:
Typically we don't inherit from the test environment we use it as an argument to types like ClientTestBase/RecorderTestBase but I see why those don't work. I'll add a LiveTestBase<TEnvironment> class today so you can use it.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/15700

Reply:
That looks kind of like IClassFixture<TEnvironment> in xUnit.

Reply:
done.